cloudmarker/events/rdbmsenforcetlsevent.py

```diff
@@ -0,0 +1,100 @@
+"""RDBMS Enforce TLS/SSL Event.
+
+This module defines the :class:`RDBMSEnforceTLSEvent` class that
+identifies RDBMS servers which have TLS/SSL connection enforcement
+disabled. This plugin works on the  properties found in the ``com``
+bucket of ``rdbms`` records.
+"""
+
+
+import logging
+
+from cloudmarker import util
+
+_log = logging.getLogger(__name__)
+
+
+class RDBMSEnforceTLSEvent:
+    """Az RDBMS TLS/SSL enforcement event plugin."""
+
+    def __init__(self):
+        """Create an instance of :class:`RDBMSEnforceTLSEvent`."""
+
+    def eval(self, record):
+        """Evaluate RDBMS servers for TLS connection enforcement.
+
+        Arguments:
+            record (dict): A RDBMS record.
+
+        Yields:
+            dict: An event record representing an RDBMS where TLS
+            connection enforcement is disabled
+
+        """
+        com = record.get('com', {})
+        if com is None:
+            return
+
+        if com.get('record_type') != 'rdbms':
+            return
+
+        ext = record.get('ext', {})
+        if ext is None:
+            return
+
+        # True, None, missing key or any other value will not
+        # genarate an event. An event will be generate only if
+        # the value of `tls_enforced` is False.
+        if com.get('tls_enforced') is False:
+            yield from _get_rdbms_tls_enforcement_event(
+                com, ext)
+
+    def done(self):
+        """Perform cleanup work.
+
+        Currently, this method does nothing. This may change in future.
+        """
+
+
+def _get_rdbms_tls_enforcement_event(com, ext):
+    """Generate event for TLS enforcement disabled.
+
+    Arguments:
+        com (dict): RDBMS record `com` bucket
+        ext (dict): RDBMS record `ext` bucket
+    Returns:
+        dict: An event record representing RDBMS with SSL
+        connection enforcement disabled
```

Comment:
The second line, i.e., "connection enforcement disabled", needs to be indented one level more. Sorry, forgot to spot this in an earlier review.

Reply:
From Google Python Style Guide section [doc function returns](https://google.github.io/styleguide/pyguide.html#doc-function-returns) no indentation is needed.
Thanks @susam  for the explanation.